### PR TITLE
Add extra dependency update steps to the README

### PR DESCRIPTION
**Pull request by teor2345**
We're missing a few dependency update steps before we do a release.

### CODE REVIEW

**Comment by teor2345** [comment]
@arya2 feel free to add the steps from PR https://github.com/ZcashFoundation/zcash_script/pull/62#pullrequestreview-1312139664 in this PR.

File: README.md [hunk 1]
```diff
@@ -24,7 +24,12 @@ to include a lot of other stuff e.g. the orchard library.
 
 1. Update `depend/zcash` with the latest tagged version of `zcashd`
 2. Update `Cargo.toml` versions to match the versions used by the latest tagged version of `zcashd`, and its dependencies
-3. Publish a new release
+3. For dependencies that are shared with Zebra (but not `zcashd`), match the latest version in Zebra's [Cargo.lock](https://github.com/ZcashFoundation/zebra/blob/main/Cargo.lock):
+    - use `cargo tree --invert <crate>` to see if the crate is from `zcash_script` or another dependency
+    - see the list in [Cargo.toml](https://github.com/ZcashFoundation/zcash_script/blob/master/Cargo.toml#L69)
+4. For new dependencies with a leading zero in their version (`0.x.y`), use a `>=` dependency [to make them automatically upgrade to match Zebra's dependencies](https://doc.rust-lang.org/cargo/reference/resolver.html#semver-compatibility)
+5. Check all open PRs to see if they can be merged before the release
+6. Publish a new release
```
Comment (arya2) [inline README.md:32]: Do we want to say run cargo release with no-publish first to commit any automatic changes?
 
```suggestion
5. Run `cargo-release` with `--no-publish` to commit any automatic changes
6. Check all open PRs to see if they can be merged before the release
7. Publish a new release
```

**Review by arya2 — DISMISSED** [review]
This looks good, I think that was the only important step missing in the README.

File: README.md (hunk 1 above)
Comment (teor2345) [reply README.md:32]: I'd like to do it in the opposite order, in case the open PRs change anything about the release or `Cargo.toml`?

**Review by arya2 — APPROVED** [review]

**Pull request merged**